### PR TITLE
s/comtrol/control

### PR DESCRIPTION
**Pull request by melcorr**
fixes #7715

Behold my outstanding contribution to docs day! 

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: docs/workflows/import-export.rst [hunk 1]
```diff
@@ -385,9 +385,8 @@ proposed import using the ``import-check`` command::
     * repo_mapping is valid JSON
 
 ``import-check`` is a low-overhead synchronous call. It does not attempt to do validations that
-require database access or long-running tasks such as verifying checksums. All parameters are optional. 
+require database access or long-running tasks such as verifying checksums. All parameters are optional.
```
Comment (mdellweg) [inline docs/workflows/import-export.rst:388]: :+1:

**Review by mdellweg — APPROVED** [review]

File: docs/workflows/import-export.rst (hunk 1 above)
Comment (melcorr) [reply docs/workflows/import-export.rst:388]: My text editor did this. I can claim no credit for this masterful removal of whitespace.

File: CHANGES/7715.doc [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix typo s/comtrol/control
```
Comment (ipanova) [inline CHANGES/7715.doc:1]: can you past tense please?

Comment (melcorr) [reply CHANGES/7715.doc:1]: Should be possible !

Comment (melcorr) [reply CHANGES/7715.doc:1]: Should be possible!!

**Review by ipanova — APPROVED** [review]

**Pull request merged**